Update api-management-howto-protect-backend-with-aad.md (#124711)
Azure/azure-content: articles/api-management/api-management-howto-protect-backend-with-aad.md

## Authorization workflow

> 1. A user or application acquires a token from Microsoft Entra ID with permissions that grant access to the backend-app. If you use the v2 endpoint, ensure that the accessTokenAcceptedVersion property is set to 2 in the application manifest of both the Azure AD Client app and the Backend app.

Reviewer: @dlepow - Azure AD is a deprecated product name. Should this be "Microsoft Entra ID" or similar instead?

Reviewer 2: @ShannonLeavitt - Good catch, we can work around this.  I suggest rewriting the highlighted sentence: "If you use the v2 token endpoint, ensure that the accessTokenAcceptedVersion property is set to 2 in the application manifest of the backend-app and any client app that you configure."

> 1. A user or application acquires a token from Microsoft Entra ID with permissions that grant access to the backend-app. If you use the v2 endpoint, ensure that the accessTokenAcceptedVersion property is set to 2 in the application manifest of both the Azure AD Client app and the Backend app.

Reviewer 3: ```suggestion
1. A user or application acquires a token from Microsoft Entra ID with permissions that grant access to the backend-app. If you use the v2 endpoint, ensure that the accessTokenAcceptedVersion property is set to 2 in the application manifest of the back end app and any client app that you configure.
```
